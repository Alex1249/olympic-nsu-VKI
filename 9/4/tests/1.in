This is a text.
Multiline text.
