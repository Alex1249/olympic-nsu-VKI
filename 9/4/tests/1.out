  ####
. ##
M #
T #
a #
e ###
h #
i ####
l ##
n #
s ##
t #####
u #
x ##
